.github/ISSUE_TEMPLATE/question.md

```diff
@@ -12,8 +12,11 @@ Please follow the issue template below. Failure to do so will result in a delay
 ## Library
 - [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
 - [ ] `@azure/msal-browser@2.x.x`
+- [ ] `@azure/msal-node@1.x.x`
+- [ ] `@azure/msal-react@0.x.x`
```

Comment:
```suggestion
- [ ] `@azure/msal-react@2.x.x`
```

Reply:
I'd be fine with starting React at `1.x`, but we can talk about that today.

Reply:
Did we come to a decision on this?